cmd/actuary/dockerconf_test.go

```diff
@@ -0,0 +1,205 @@
+package actuary
+
+
+import (
+
+	"encoding/json"
+	"testing"
+	"github.com/docker/engine-api/types"
+)
+
+//2. Docker daemon configuration
+
+//This function is used in all functions that call getProcCmdLine -- replacing the output of getProcCmdLine for the pass case and fail case
+func procCmdLineHelper(t *testing.T, orig func(t Target) Result, procPass []string, procFail []string, err1 string, err2 string){
+
+	procFunction = func(procname string) (cmd []string, err error){
+			err = nil
+			cmd = procPass
+			return
+		}
+
+	for i := 0; i< 2; i++{
```

Comment:
We probably shouldn't have loops that iterate twice. Maybe use a `case` here to replace the `for`?